### DISCUSSION TITLE
is there any short plan to implement Compression in DuckDB storage format

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#4299] [General]
just imported a couple of parquet files into DuckDB native storage, the data went from 2.6 GB to 16 GB, is there any plan to implement compression in the short term.

**Comment by Alex-Monahan** [comment]
Compression is actively being improved. Was this on the latest master branch? Any chance you could provide the files or anonymized versions of them?

**Comment by djouallah** [comment]
using the latest master branch, you can download the files here
https://drive.google.com/drive/folders/1mZC3NuPBZC4mjP3_kH18c9fLrv8ME7RU

**Reply by Mytherin** [reply]
As Alex mentioned, compression is actively being worked on. On the latest master I get `7.6GB`, not `16GB`. Expect more improvements on that in the near future.

**Comment by djouallah** [comment]
maybe the index is taking more space

https://github.com/djouallah/Testing_BI_Engine/blob/main/Database/DuckDB/Create_Empty_Tables.py

https://github.com/djouallah/Testing_BI_Engine/blob/main/Database/DuckDB/Create_db_import.py

**Reply by Mytherin** [reply]
Yes, that is probably it. The ART indexes are not compressed yet. I would recommend not creating indexes if you need to conserve space.
